{% extends "base.html" %}
{% block content %}

<!-- Slideshow -------------------------------------------------------------->
<div class="w3-content w3-display-container" style="width:100%">
  <div class="w3-display-container mySlides">
    <img src="{{ url_for ('static', filename='images/oat.JPG') }}" style="width:100%">
    <div class="w3-display-middle w3-large w3-container w3-padding-16 w3-black">
    Oat
    </div>
  </div>

  <div class="w3-display-container mySlides">
    <img src="{{ url_for ('static', filename='images/wheat.jpg') }}" style="width:100%">
    <div class="w3-display-middle w3-large w3-container w3-padding-16 w3-black">
    Wheat
    </div>
  </div>
  <div class="w3-display-container mySlides">
    <img src="{{ url_for ('static', filename='images/barley.JPG') }}" style="width:100%">
    <div class="w3-display-middle w3-large w3-container w3-padding-16 w3-black">
    Barley
    </div>
  </div>


  <button class="w3-button w3-black w3-display-left" onclick="plusDivs(-1)">&#10094;</button>
  <button class="w3-button w3-black w3-display-right" onclick="plusDivs(1)">&#10095;</button>
</div>

 <!-- About Section ---------------------------------------------------------->
<div class="w3-content" style="width:70%">
  <div class="w3-row w3-padding-64" id="about">
    <div class="w3-col m6 w3-padding-large w3-grey" style="overflow-y:scroll; height:500px">
      <h1 class="w3-center">About</h1><br>
      <p class="w3-medium">Full genome sequences of wheat, barley, and oat will accelerate the improvement of these important Canadian cereal crops. However, a complete genome sequence is only a single 'dimension' in the overall picture of genome science. In this metaphor, the second dimension is sequence diversity, and the third could be epigenetic factors: modifications to DNA that influence gene expression. These three dimensions conspire to determine gene expression over time, and the diversity of gene expression in a species. In this project, we propose to create a foundational resource that embraces this multidimensional complexity, upon which new discoveries and innovations in Canadian cereal crops will be built.</p>
      <p class="w3-medium">This work will employ a strategy of exome capture, whereby a nearly-complete gene-space can be re-sequenced in many different genetic accessions. Through this strategy, we will interrogate gene diversity in cultivated germplasm and related wild species of wheat, oat, and barley. The first will provide information about single nucleotide polymorphisms (SNPs) and small insertion-deletions (indels) in the germplasm of Canadian breeding programs, while DNA variants from wild species can be used to identify new alleles, providing opportunities to identify and validate targets for crop improvement through introgression or gene editing. This work will also provide rich opportunities to refine phylogenetic inferences about the origins and evolution of cereal crops.</p>
      <p class="w3-medium">Until recently, the analysis of epigenetic effects has been difficult. Now, with the availability of a complete wheat sequence, and with refinements in the available tools, we have planned some of the first careful experiments to address some epigenetic-related hypotheses. Our work will be based on the analysis of DNA modifications among a series of wheat ancestors and synthetic polyploids, focusing specifically on vernalization-dependent flowering time. The data from wheat exome sequences will be extended to include 'skim' re-sequencing, bisulfite sequencing, chromatin immuno-precipitation, micro RNA sequencing, and RNASeq. These approaches will allow us to discover how DNA modifications affect gene expression following genome polyploidization: information that could be applied in crop improvement.</p>
      <p class="w3-medium">All of our work will be unified in three parallel databases, each built around the best available sequence from the cultivated genomes of wheat, oat, and barley. New data analysis tools will include automated SNP discovery and annotation, as well as algorithms to assign allele variants from wild species to the cultivated references. The databases will facilitate allele mining from candidate genes and/or specific genetic intervals, and we will test this opportunity through an in-depth analysis of candidate genes affecting flowering, fungal resistance, and seed fibre. A user-friendly interface and data export tools will disseminate these results to breeders and other end-users. For example, users will identify SNPs based on their position on the reference sequence and their polymorphism in mapping or breeding parents. In this way, they can plan targeted assays with a high probability of successful deployment.</p>
      <p class="w3-medium">Although the scope of this work is ambitious, our team is confident that we now have the right tools and the right expertise to generate these outcomes, and that the work will benefit from our ability to implement efficient parallel approaches in three crops. Furthermore, this parallel work will afford the best possible opportunities to explore and exploit comparative genomics. The resource that we build will be immediately useful to Canadian plant breeders and genomic researchers, as well as to our valuable international collaborators.</p>		
    </div><!-- About -->

    <div class="w3-col m6 w3-padding-large w3-hide-small w3-light-grey" style="overflow-y:scroll; height:500px">
      <h1 class="w3-center">News</h1><br>
      {% if session.logged_in %}
        <a href="{{ url_for("news_add") }}" class="w3-button w3-large w3-blue">Add</a>
        <a href="{{ url_for("news_delete") }}" class="w3-button w3-large w3-red">Delete</a>
      <br/><br/>
      {% endif %}

      {% for x in news %}
        <p class = "w3-medium"><b> {{ x.date }} </b></p>
        <p class = "w3-medium"> {{ x.entry }} </p>
        <br>
        {% endfor %}
    </div> <!-- News -->

  </div> <!-- Row -->
</div> <!-- Outer div -->

<script>
var slideIndex = 1;
showDivs(slideIndex);

function plusDivs(n) {
  showDivs(slideIndex += n);
v}

function showDivs(n) {
  var i;
  var x = document.getElementsByClassName("mySlides");
  if (n > x.length) {slideIndex = 1}    
  if (n < 1) {slideIndex = x.length}
  for (i = 0; i < x.length; i++) {
     x[i].style.display = "none";  
  }
  x[slideIndex-1].style.display = "block";  
}
</script>
{% endblock %}
